src/DSP/MoogFilter.h

```diff
@@ -0,0 +1,53 @@
+/*
+  ZynAddSubFX - a software synthesizer
+
+  Moog Filter.h - Several analog filters (lowpass, highpass...)
+  Copyright (C) 2018-2018 Mark McCurry
+  Author: Mark McCurry
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+*/
+
+#pragma once
+#include "Filter.h"
+
+namespace zyn {
+
+class MoogFilter:public Filter
+{
+    public:
+        //! @param Fq resonance, range [0.1,1000], logscale
+        MoogFilter(unsigned char Ftype, float Ffreq, float Fq,
+                unsigned int srate, int bufsize);
+        ~MoogFilter() override;
+        void filterout(float *smp) override;
+        void setfreq(float /*frequency*/) override;
+        void setfreq_and_q(float frequency, float q_) override;
+        void setq(float /*q_*/) override;
+        void setgain(float dBgain) override;
+        void settype(unsigned char type); //
+
+    private:
+        unsigned sr;
+        float gain;
+        unsigned char type;
```

Comment:
Type appears to be a write only variable. The only place I see it used is in setType(). So, you should be able to remove it without a problem. Pretty minor all things considered as saving a byte in a full zyn instance is basically nothing :p